bin/melody

```diff
@@ -1,13 +1,19 @@
 #!/usr/bin/env php
 <?php
 
-if (!file_exists(__DIR__.'/../vendor/autoload.php')) {
-    echo "Melody's dependencies not installed.\n";
+if (file_exists($autoloader_path = __DIR__.'/../vendor/autoload.php')) {
+    require_once $autoloader_path;
+} elseif (file_exists($autoloader_path = __DIR__.'/../../../vendor/autoload.php')) {
```

Comment:
this is the wrong path to handle the case of melody being installed as a dependency. You should not move outside the vendor folder and then inside it again, because this would be incompatible with renamed vendor dirs